### PR TITLE
Build and test for Javacpp-developerToolsServer API

### PR DESCRIPTION
**Pull request by jbkyang-nvi**
Add build script and testing for Java bindings. Cleaning for generic_server_wapper.
Javacpp changes: https://github.com/bytedeco/javacpp-presets/pull/1361
Developer tools test: https://github.com/triton-inference-server/developer_tools/pull/20
Client script PR: https://github.com/triton-inference-server/client/pull/331
Server README PR: https://github.com/triton-inference-server/server/pull/5883

### CODE REVIEW

File: java-api-bindings/install_dependencies_and_build.sh [hunk 1]
```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (dyastremsky) [inline java-api-bindings/install_dependencies_and_build.sh:2]: 2023

File: java-api-bindings/install_dependencies_and_build.sh [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+Installs Maven, Java JDK and builds Tritonserver Java bindings
+-h|--help                  Shows usage
+-t|--triton-home           Expected Trition library location, default is: /opt/tritonserver
+-b|--build-home            Expected build location, default is: /tmp/build
+-v|--maven-version         Maven version, default is: "3.8.4"
+-c|--core-tag              Tag for core repo, defaut is: "main"
+-j|--jar-install-path      Path to install the bindings .jar
+--javacpp-branch           Javacpp-presets git path, default https://github.com/bytedeco/javacpp-presets.git
+--javacpp-tag              Javacpp-presets branch tag, default "master"
```
Comment (dyastremsky) [inline java-api-bindings/install_dependencies_and_build.sh:38]: "default is: " for javacpp-branch and javacpp-tag for consistency with above. Or any format, would just be good to keep it consistent. Also a typo on "default" for core-tag.

File: java-api-bindings/install_dependencies_and_build.sh [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+# Build static libraries
+## install cmake and rapidjson
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | \
+      gpg --dearmor - |  \
+      tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null && \
+      apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' && \
+      apt-get update && \
+      apt-get install -y --no-install-recommends \
+        cmake-data=3.21.1-0kitware1ubuntu20.04.1 \
```
Comment (dyastremsky) [inline java-api-bindings/install_dependencies_and_build.sh:120]: Could be good to move CMake version to a variable at the top.

**Review by dyastremsky — APPROVED** [review]

File: qa/L0_java_simple/test.sh [hunk 4]
```diff
@@ -0,0 +1,84 @@
+
```
Comment (krishung5) [inline qa/L0_java_simple/test.sh:1]: nit: I think the line `#!/bin/bash` is missing?

**Review by krishung5 — APPROVED** [review]

**Pull request merged**